snap/local/files/run_dcgmproftester.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Please add some comments to this script for future readers. :) It would be good to include background information such as why some dcgproftesterXX may be missing on some platforms.

Reply:
yeah that's a good point. It's not obvious that due to limitations in snapcraft we cannot have a single snapcraft.yaml for all architectures given that dcgmproftester10 does not exist for ARM - this wrapper is just a workaround

Reply:
Resolving, the comments were added, thanks!